### ISSUE TITLE
Initialize in repo with pre-existing commit hooks

### ISSUE THREAD

**Issue by chiptopher** [chiptopher/guet#1]
## Feature Request
When initializing guet in a repo that already has a pre-commit hook, put it both in a sub folder and replace it with a script that calls both.

#### Context
`guet` is unusable for projects that already have a pre-commit hook.

#### Acceptance Criteria
When there is a folder structure like this:
```
/.git
  |- /hooks
       | - pre-commit
```
calling `guet start` will produce something like this:
```
/.git
  |- /hooks
       | - pre-commit
       | - /pre-commit-hooks
             | - guet-pre-commit
             | - old-pre-commit
```

**Comment by chiptopher** [comment]
The pre-commit hook should call guet-pre-commit hook and the old-pre-commit hook.

**Comment by chiptopher** [comment]
Another thought: maybe have the user decide if they want to overwrite the current pre-commit hook or just put the file in the hooks folder and let them do it.

**Comment by chiptopher** [comment]
Closed by #10
